easybuild/easyconfigs/p/ParaView/ParaView-4.4.0-gimkl-2.11.5-mpi.eb

```diff
@@ -27,6 +27,7 @@ dependencies = [
     ('libXext', '1.3.3'),
     ('libX11', '1.6.3'),
     ('zlib', '1.2.8'),
+    ('HDF5', '1.8.16', '-serial', ('GCC', '4.9.3')),
```

Comment:
I agree, but since you're running into a conflict, it's a problem.

We're considering to switch to having `--minimal-toolchains` enabled by default in EasyBuild v3.0... (cfr. http://easybuild.readthedocs.io/en/latest/Manipulating_dependencies.html#using-minimal-toolchains-for-dependencies)